packages/Toast/src/Toast.styles.js

```diff
@@ -59,8 +59,7 @@ export const IconStyled = styled.div`
   color: ${({ kind }) => iconColors[kind]};
   flex-grow: 0;
   flex-shrink: 0;
-  padding-right: ${tokens.space};
-  padding-top: 1px;
+  margin: 1px ${tokens.space} 0 0;
```

Comment:
Not sure why the icon doesn't shrink in Paprika's Storybook, but in the app, the full width content of the Toast was causing the icon to shrink quite a lot.